docs/docsite/Makefile

```diff
@@ -44,6 +44,7 @@ webdocs: docs
 
 #TODO: leaving htmlout removal for those having older versions, should eventually be removed also
 clean:
+	-rm -rf $(BUILDDIR)
```

Comment:
Hrm- this is an awfully big penalty to impose on someone that accidentally sets `BUILDDIR` to `.` or `/`...

Reply:
I agree, and I waffled about including the clean...